### ISSUE TITLE
Beginner difficulty text does not meet color contrast standard

### ISSUE THREAD

**Issue by flautan** [flautan/prova#1]
Element has insufficient color contrast of 3.3 (foreground color: #1185d7, background color: #eaecee, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
